docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -19,6 +19,82 @@ If applicable, exact error messages linked to this issue so users searching for
 Link to fix
 ////
 
+*Upgrading to v8.15.0 may cause APM indices to lose their lifecycle policy* +
+_Elastic Stack versions: 8.15.0_ +
+_Fixed in Elastic Stack version 8.15.1_
+
+// The conditions in which this issue occurs
+The issue only occurs when _upgrading_ the {stack} to 8.15.0.
+The issue does _not_ occur when creating a _new_ cluster using 8.15.0.
+The issue also does not occur if a custom ILM policy is configured using a custom component template.
+
+// Describe why it happens
+In 8.15.0, APM Server switched to use Data stream Lifecycle to manage data retention
```

Comment:
Based on the [Elasticsearch docs that cover this topic](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-lifecycle.html), I think all three words should be lowercase: data stream lifecycle.

```suggestion
In 8.15.0, APM Server switched to use data stream lifecycle to manage data retention
```